src/Test/Utilities/Portable/TestBase.cs

```diff
@@ -362,9 +362,24 @@ private static MetadataReference GetOrCreateMetadataReference(ref MetadataRefere
 
         protected static string NormalizeDiagnosticString(string inputString)
         {
-            if (!inputString.Contains("\r\n") && inputString.Contains("\n"))
+            if (inputString == null)
             {
-                return inputString.Replace("\n", "\r\n");
+                return inputString;
+            }
+
+            if (ExecutionConditionUtil.IsWindows)
+            {
+                if (!inputString.Contains("\r\n") && inputString.Contains("\n"))
+                {
+                    return inputString.Replace("\n", "\r\n");
+                }
+            }
+            else
+            {
+                if (inputString.Contains("\r\n"))
```

Comment:
`if` check is not necessary.

Reply:
Will fix that up in the upcoming PR to enable Emit tests.